packages/eslint-plugin-stories/src/rules/stories-default-export.ts

```diff
@@ -11,27 +10,28 @@ const rule: Rule.RuleModule = {
     }
 
     return {
-      ObjectExpression(node) {
-        if (node.parent.type !== 'ExportDefaultDeclaration') {
+      ExportDefaultDeclaration(node) {
+        hasDefaultExport = true;
+
+        if (node.declaration.type !== 'ObjectExpression') {
```

Comment:
Noticed that this rule could be simplified.

Previously it looked at all `ObjectExpression`s and determined if any were an `ExportDefaultDeclaration`. Not sure why I wrote it like that.

Now it reverses that. First look for an `ExportDefaultDeclaration`, then determine if it's an object.